backend/apid/graphql/silenced.go

```diff
@@ -57,10 +54,10 @@ func (r *silencedImpl) Check(p graphql.ResolveParams) (interface{}, error) {
 	return handleControllerResults(r.checkFinder.Find(ctx, sil.Check))
 }
 
-// Environment implements response to request for 'environment' field.
-func (r *silencedImpl) Environment(p graphql.ResolveParams) (interface{}, error) {
-	return findEnvironment(p.Context, r.envFinder, p.Source.(types.MultitenantResource))
-}
+// Namespace implements response to request for 'namespace' field.
+// func (r *silencedImpl) Namespace(p graphql.ResolveParams) (interface{}, error) {
```

Comment:
The code should be uncommented, or removed, I think.

Reply:
cc @jamesdphillips

Reply:
Kill it